### ISSUE TITLE
Auto download new chapters

### ISSUE THREAD

**Issue by AleksiejLublov** [xgi/houdoku#242]
### Acknowledgements

- [X] Check that there is not an existing ticket for this request, and that it is not related to an existing ticket.
- [X] Create one ticket per feature. Feel free to create multiple tickets.
- [X] If this is an extension-related request, open a ticket in the [extensions repo](https://github.com/xgi/houdoku-extensions/issues/new/choose) instead.
- [X] Write a concise title for this ticket in the field above.

### Feature details

Basically as the title says, it would be awesome if future releases had option to automatically download new found chapters after refreshing the library.
